provision/nginx/conf.d/dispatcher.rb

```diff
@@ -104,8 +111,9 @@ def initialize(ip, port, haco, env = [], hostname = '')
       @haco = haco
 
       @root = '/var/lib/haconiwa'
-      @id = "#{@haco}-#{@ip.gsub('.', '-')}"
-      @environment = ["IP=#{@ip}", "PORT=#{@port}", "ID=#{@id}"]
+      @id = "#{@haco}-#{@ip.gsub('.', '-')}-#{Time.now.to_i}"
+      # @id = "#{@haco}-#{@ip.gsub('.', '-')}-1540895210"
+      @environment = ["IP=#{@ip}", "PORT=#{@port}", "ID=#{@id}", "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"]
```

Comment:
I see.

Reply:
This `PATH` will be removed when this is supported in haconiwa side.